towncrier.toml

```diff
@@ -0,0 +1,37 @@
+[tool.towncrier]
+directory = "changelog.d"
```

Comment:
I would very much like to use a different directory name here. Two ideas:
* jobbergate-changelog -- follows subproject naming convention
* .changelog           -- makes it less intrusive
* changes              -- simple and straightforward

Reply:
Renamed it to `changes`: [f327fc5](https://github.com/omnivector-solutions/jobbergate/pull/690/commits/f327fc5bb2ec4eb190b082957f6d454fd385a6c8).